src/elements/circular-percentage-bar/src/percentage-bar-styles.ts

```diff
@@ -6,63 +6,49 @@ export const CircularPercentageBarStyles = css({
     float: 'left',
     width: '1em',
     height: '1em',
-    'font-size': '120px',
-    '-webkit-border-radius': '50%',
-    '-moz-border-radius': '50%',
-    '-ms-border-radius': '50%',
-    '-o-border-radius': '50%',
-    'border-radius': '50%'
+    fontSize: '120px',
+    borderRadius: '50%'
   },
   '.c100.xs': {
-    'font-size': '64px'
+    fontSize: '64px'
   },
   '.c100.sm': {
-    'font-size': '80px'
+    fontSize: '80px'
   },
   '.c100.md': {
-    'font-size': '120px'
+    fontSize: '120px'
   },
   '.c100.lg': {
-    'font-size': '160px'
+    fontSize: '160px'
   },
   '.c100.xl': {
-    'font-size': '200px'
+    fontSize: '200px'
   },
   '.c100 .bar, .c100 .bar-full, .c100.gt50 .fill, .c100 .fill-full': {
     position: 'absolute',
     border: '0.08em solid #307bbb',
     width: '0.84em',
     height: '0.84em',
     clip: 'rect(0em, 0.5em, 1em, 0em)',
-    '-webkit-border-radius': '50%',
-    '-moz-border-radius': '50%',
-    '-ms-border-radius': '50%',
-    '-o-border-radius': '50%',
-    'border-radius': '50%'
+    borderRadius: '50%'
   },
   '.c100.gt50 .bar:after, .c100.gt50 .fill, .c100 .bar-full:after, .c100 .fill-full': {
-    '-webkit-transform': 'rotate(180deg)',
-    '-moz-transform': 'rotate(180deg)',
-    '-ms-transform': 'rotate(180deg)',
-    '-o-transform': 'rotate(180deg)',
```

Comment:
Did we check on the other browsers?

Reply:
this should be added automatically with theme-ui. I've tried it on browserstack and it worked for me, but If somebody can try it on an actual machine it would be good (I'm on macOS so I don't have internet explorer). Tested on chrome, Opera, Mozzila and Safari.

Reply:
It should be fine then.